tests/integration_tests/mechanics/debug_mode/tests

```diff
@@ -7,6 +7,11 @@
       output = 'Debug_Run_o/dispatch.nc'
       gold_files = 'gold/dispatch.nc'
     [../]
+    [./debug_plot]
+      type = NetCDF
+      output = 'Debug_Run_o/dispatch_id0_y10_c0.png'
+      gold_files = 'gold/dispatch_id0_y10_c0.png'
```

Comment:
I don't think the output needs a gold file, since it just checks generation, but I can see what this does later.